guides/common/modules/proc_performing-a-full-backup.adoc

```diff
@@ -8,6 +8,9 @@ There are two main methods of backing up {ProjectServer}:
 * Offline backup
 * Online backup
 +
+With online backups, only services that affect the consistency of the backup are shut down while the backup process is running.
+For production usecases, use online backups. 
```

Comment:
So, when should I use offline ones? ;-)

Reply:
@evgeni should I remove this? I wasn't sure about this

Reply:
I would, yeah.

Both backup methods are fine for production usecases, that was the point of the changes we implemented.

The tradeoff for the user is:
- When doing an offline backup, all services are down and we can just assume the data copied is consistent (= we perform no checks)
- When doing an online backup, only some services (which we know to produce inconsistencies) are down. As there still can be something interfering with the consistency, we perform checks (and may restart data-copy operations) which take additional time.